src/elements/TabbedContainer/Tab.js

```diff
@@ -0,0 +1,31 @@
+import './TabbedContainer.less';
```

Comment:
This is already imported by the `TabbedContainer` component. This should either import a specific `Tab.less`, or not import a less at all.